.github/workflows/main.yml

```diff
@@ -70,6 +70,35 @@ jobs:
           file: build/*/*.dmg
           tag: ${{ github.ref }}
           file_glob: true
+          
+  appcast:
+    name: Set Appcast
+    needs: build
+    runs-on: macos-latest
+    env:
+      JOB_TYPE: BUILD
+      SIGNATURE_FOR_SIGNING: ${{ secrets.SIGNATURE_FOR_SIGNING }}
+    if: github.event_name == 'release'
+    steps:
+      - uses: actions/checkout@v2
+      
+      - name: Get Tag
+        run: echo "TAG_VER=${GITHUB_REF/refs\/tags\//}" >> $GITHUB_ENV
+      
+      - name: Get Information & Sign
+        run: |
+           echo "DATE=$(date -R)" >> $GITHUB_ENV
+           echo "SIGNATURE=$(./bin/sign_update -s ${{ secrets.SIGNATURE_FOR_SIGNING }} build/Release/MaciASL-${{ env.TAG_VER }}-RELEASE.dmg)" >> $GITHUB_ENV
+           ./bin/generate_appcast -o appcast.xml -s ${{ secrets.SIGNATURE_FOR_SIGNING }} --download-url-prefix https://github.com/acidanthera/MaciASL/releases/download/${{ env.TAG_VER }}/ build/Release
+      
+      - name: Commit Appcast
+        run: |
+          git config --local user.email "41898282+github-actions[bot]@users.noreply.github.com"
```

Comment:
What is 41898282? Where does this come from?

Reply:
41898282 can be found in GitHub API: https://api.github.com/users/github-actions%5Bbot%5D

Reply:
Leave a comment on this, because it feels kind of confusing.

Reply:
I guess this not necessary. However, if I recall correctly, the person how commits would be who did the release. I could remove it though

Reply:
Well, it makes sense to show that the appcast is committed by a bot. Just I want an explanation on what the bot is like "The 41898282 identifier comes from the GitHub Actions API: https://api.github.com/users/github-actions%5Bbot%5D."

Reply:
tbh, I just looked what the e-mail address for the GitHub actions bot is and pasted it there.